helm/kagent-crds/templates/kagent.dev_modelconfigs.yaml

```diff
@@ -8,8 +8,6 @@ metadata:
 spec:
   group: kagent.dev
   names:
-    categories:
```

Comment:
not sure how this happened. i'll check

Reply:
I always get this as well when building locally